cargo-dist/src/lib.rs

```diff
@@ -209,6 +209,7 @@ pub fn fetch_updater_from_source(dist_graph: &DistGraph, updater: &UpdaterStep)
     // Install to a temporary path before moving it to the destination
     cmd.arg("install")
         .arg("axoupdater-cli")
+        // .arg("--features=tls_native_roots")
```

Comment:
Is this meant to be uncommented once we've shipped an axoupdater change?

Reply:
Ack, yes! and it has shipped